.github/workflows/cd.yml

```diff
@@ -302,12 +302,16 @@ jobs:
           registry-url: "https://registry.npmjs.org"
       - name: Publish the package
         shell: bash
+        working-directory: npm/git-cliff
         run: |
-          cd npm/git-cliff
           yarn install
           yarn build
           cp ../../README.md .
-          npm publish --access public
+          if [ ${{ contains(github.ref, '-') }} = "true" ]; then
+            yarn npm publish --tag rc
```

Comment:
Ah, one question: is this going to tag the release as "rc"? If so, what happens if it tries to overwrite it?

Reply:
If by overwrite you mean a new RC version then the new version gets assigned the tag and it's unassigned from the previous one. In fact, under the hood the lack of --tag sets the tag to "latest" so the same mechanism applies for that as for rc.

Reply:
got it, thanks for the explanation!